teerex/sgx-verify/fuzz/Cargo.lock

```diff
@@ -0,0 +1,3445 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This cargo lock can be removed if you add the `sgx-verify-fuzz` to the workspace.

Reply:
The setup was automatically created by `cargo-fuzz` and they create a workspace on purpose. I'm not sure how this would affect their CLI-tools. Therefore I think I would stick to their setup. Or do you have a strong opinion?
```
# Prevent this from interfering with workspaces
[workspace]
members = ["."]
```

Reply:
Usually, I have, but I guess they will have their reasons. So we can leave it like that.